### DISCUSSION TITLE
Welcome to Token Magic FX Discussions!

### DISCUSSION THREAD

**Discussion by Feu-Secret** [Feu-Secret/Tokenmagic#72] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adamblak** [comment]
So I'm getting an effect on tokens that I'd like to be able to toggle off?  If you set the height of a token to something very high, say 1-5k feet, the token ends up swirling into complete distortion.  I think this was a setting which at smaller heights ~50-100ft makes it seem hovering high above everything else with a slight sway.  But at massively increased heights, its breaking the token's appearance completely.  I've looked through all of the preference panes, and I couldn't find a way to toggle this animation on or off, but maybe I'm missing something somewhere?  

My use case is, I have an aerial battle happening about a mile up.  So let's say about 5k feet.  I'd like for combat purposes to be able to track overall height position semi-accurately, as falling or diving might be part of the combat, so I'd like to be able to update the height in semi accurate increments, but I'd also like to keep Token Magic enabled! 

Example pic of what I'm seeing:

<img width="397" alt="Distortion effect" src="https://user-images.githubusercontent.com/38922243/129917737-ee04af3b-aa37-420f-b1ec-f51fd7ff759f.png">

**Reply by adamblak** [reply]
Is there a better place to try to resolve this??  Is there a discord or something where this might be more appropriate to discuss!

**Comment by LeonxnoeL** [comment]
Any plans to update the module to Foundry version 11? I miss this module a lot when using v11. It is pretty good.

**Comment by Killem2** [comment]
Hey there I don't know if this is the right place but I really do love this module and I had some questions I was trying to edit some of the default scripts to see if I could make it act a certain way I wanted to be able to take some of those effects and apply them to a token but also make it look like an aura and I've gotten pretty close but I think what's happening is that I don't really know enough about this to get it to do it right. 

So the objective I was trying to do was have my token and let's just say for example I use the evil aura right for an antipaladin so I can see how far his antipaladin aura goes out but I wanted to look you know menacing and evil when I do it by default it just basically colors the token and that's about all it does I played around with a little bit and got it to where it doesn't colorize the token as much and it kind of becomes an aura but it's only out like 5 to maybe something like 10 ft on the character and when I play around with the things that look like they should be the radius or at least to me it looks like it should be something that would increase the distance that this graphic goes out it goes out a little bit further but then it just becomes square and I'm not really sure what to do. 

So is trying to figure out if there's already logic built in that I'm just not putting in correctly or if this is just not a capability of this module
